openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -1087,7 +1087,7 @@ A successful Batch Credential Response MUST contain all the requested Credential
 
 The following parameters are used in the Batch Credential Response:
 
-* `credential_responses`: REQUIRED. Array that contains Credential Response objects, as defined in (#credential-request), and/or Deferred Credential Response objects, as defined in (#deferred-credential-request). Every entry of the array corresponds to the Credential Request object at the same array index in the `credential_requests` parameter of the Batch Credential Request.
+* `credential_responses`: REQUIRED. Array that contains Credential Response objects, as defined in (#credential-response), and/or Deferred Credential Response objects, as defined in (#deferred-credential-response), except `c_nonce` and `c_nonce_expires_in`. Every entry of the array corresponds to the Credential Request object at the same array index in the `credential_requests` parameter of the Batch Credential Request.
```

Comment:
```suggestion
* `credential_responses`: REQUIRED. Array that contains Credential Response objects, as defined in (#credential-response), and/or Deferred Credential Response objects, as defined in (#deferred-credential-response), with the exception of `c_nonce` and `c_nonce_expires_in`. Each element within the array matches the corresponding Credential Request object by array index in the `credential_requests` parameter of the Batch Credential Request.
```

Reply:
Suggested change  by @peppelinux  LGTM

Reply:
LGTM too. will merge this PR when this suggestion is incorporated.

Reply:
Done Kristina. Please check